t/config-center-yaml/secret.t

```diff
@@ -61,7 +61,7 @@ __DATA__
 secrets:
   - id: vault/1
     prefix: kv/apisix
-    token: root
+    token: "$ENV://VAULT_TOKEN"
```

Comment:
So we can no longer use such a configuration?
```
token: root
```

Reply:
Yes, we won't be able to use such a configuration.